QuizApp/Data/Quiz/QuizRepository.swift

```diff
@@ -43,4 +60,12 @@ class QuizRepository: QuizRepositoryProtocol {
         return QuizSessionResultRepoModel(from: quizSessionResultResponse)
     }
 
+    private func fetchedRecently() -> Bool {
+        guard abs(lastFetchedTime.timeIntervalSinceNow) > 120 else {
+            return true
+        }
+
+        return false
+    }
```

Comment:
This can be a computed property, 120 a constant and avoid evaluating boolean expressions and then returning hardcoded `true` and `false`.